src/operator/nn/fully_connected-inl.h

```diff
@@ -47,7 +48,23 @@ namespace fullc {
 enum FullyConnectedOpInputs {kData, kWeight, kBias};
 enum FullyConnectedOpResource {kTempSpace};
 enum FullyConnectedOpOutputs {kOut};
-}  // fullc
+enum FullyConnectedGradGradOutputs {
+  k_o_y_grad,
```

Comment:
please use the same camelCase to be consistent.

Reply:
It's quite unreadable for this purpose, but ok.